ChangeLog.md

```diff
@@ -1,5 +1,153 @@
 # Changelog for haskell-language-server
 
+## 1.3.0
+
+2021 July release of HLS arrives! This release includes binaries for GHC 9.0.1
+and some new interesting features. Here is the brief summary of changes:
+
+- Binaries for GHC 9.0.1 are added by @anka-213.
+- Call hierarchy plugin is added, contributed by @July541.
+  ![hierarchy](https://user-images.githubusercontent.com/12473268/127550041-094151a6-be7b-484a-bb82-c61f326ca503.gif)
```

Comment:
Now, I notice that call hierarchy does not work when execute it on a type signature. Is this intended, @July541?

Reply:
Yes, reference other languages' implementations, it doesn't belong to call hierarchy I think.